docs/developer-tools/runtime/runtime-protocol-reference/marlowebulksync-subprotocol.md

```diff
@@ -0,0 +1,55 @@
+---
+title: MarloweBulkSync sub-protocol
+sidebar_position: 3
+---
+
+The MarloweBulkSync sub-protocol is defined here: 
+
+- **[https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe-runtime/sync-api/Language/Marlowe/Protocol/BulkSync/Types.hs](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe-runtime/sync-api/Language/Marlowe/Protocol/BulkSync/Types.hs)**
+
+The MarloweBulkSync sub-protocol is used to synchronize all marlowe contract transactions. 
+The client receives a stream of blocks, each of which contains collections of create, apply inputs, and withdraw transactions.
+
+This is a firehose protocol. It gives far more information than the header sync
+or regular sync protocols, but for following a small set of contracts may be an
+inefficient choice.
```

Comment:
```suggestion
This is a firehose protocol. It gives far more information than the header sync
or regular sync protocols, but it may be inefficient for following a small set of contracts.
```